docs/data/joy/getting-started/overview/overview.md

```diff
@@ -2,43 +2,57 @@
+We're striving for the essential only, both in the component API and design (look & feel).
+Components should have only what they need to do the job.
+What is considered essential will be drawn from MUI's experience over the years developing component libraries like this one, as well as from benchmarks of modern API and design guidelines—especially when it comes to developing web apps.
 
 ### Looks great out-of-the-box
 
-Joy needs to be simple yet characteristic. Visual attributes such as scale, size, and density should be consistent across all of the components so they live together nicely. We aim to spark delight with simplicity and attention to detail. You should feel like your UI looks great from the start.
+Joy UI needs to be simple yet characteristic.
+Visual attributes such as scale, size, and density should be consistent across all of the components so they live together nicely.
+We aim to spark delight with simplicity and attention to detail.
+You should feel like your UI looks great from the start.
 
 ### Encourage creativity
 
-We're aiming for Joy to be entirely customizable, and seen as a great starting point. This is meant to encourage you to extend, change, and revamp how Joy looks. Be creative by making it your own.
+We're aiming for Joy UI to be entirely customizable, and seen as a great starting point.
+This is meant to encourage you to extend, change, and revamp how Joy UI looks.
+Be creative by making it your own.
 
 ### Focus on developer experience
 
-A great developer experience is not only the quality of the code we ship but also how clear the documentation is, and what learning resources are available for developers. We hope that bundling it all together brings joy.
+A great developer experience is not only the quality of the code we ship but also how clear the documentation is, and what learning resources are available for developers.
+We hope that bundling it all together brings joy.
 
 ## Joy UI vs. Material UI
 
-Joy UI is meant to feature the same list of components you'd find in Material UI, with similar philosophy around component API and customization extensibility, but without the Material Design implementation.
+Joy UI is meant to feature the same list of components you'd find in Material UI, with similar philosophy around component API and customization extensibility, but without the Material Design aesthetic.
 
-If you ever wanted to use Material UI for the breadth of supported components, carefully written component API, and reliability of a tried and tested library but hesitated because of Material Design, Joy UI is here to solve that for you and act as the best alternative for that.
+If you ever wanted to use Material UI for the breadth of supported components, carefully written component API, and reliability of a tried and tested library but hesitated because of Material Design, Joy UI is here to solve that for you and act as the best alternative.
```

Comment:
I have always interpreted "that" as having a negative connotation. Now it could also make sense, people not linking Material Design, would also refer to it as "that", not "this".